src/j1client.js

```diff
@@ -90,10 +90,21 @@ class JupiterOneClient {
     let results = [];
 
     while (!complete) {
+      // is this a good syntax for enums in JS? If so I'll move it out of the function's scope.
+      const JobStatus = {
+        IN_PROGRESS: "IN_PROGRESS",
+        COMPLETED: "COMPLETED",
+        FAILED: "FAILED"
+      };
       const query = gql`
```

Comment:
It would be better to pull this graphql query to top-level const and make `query` a variable that is passed in.

Reply:
changed to a function taking arguments `j1ql` and `page`